openmc/checkvalue.py

```diff
@@ -1,3 +1,17 @@
+from collections import Iterable
+from numbers import Integral
+
+import numpy as np
+
+def _isinstance(value, expected_type):
+    """A Numpy-aware replacement for isinstance"""
+    np_ints = [np.intc, np.intp, np.int8, np.int16, np.int32, np.int64,
+               np.uint8, np.uint16, np.uint32, np.uint64]
+    if expected_type is Integral:
+        types = np_ints + [Integral]
+        return any(isinstance(value, t) for t in types)
```

Comment:
Whoops, hadn't noticed that `isinstance` took tuples.  I've added some code so that `_isinstance` will handle tuples as well.